samples/bluetooth/central_uart/src/main.c

```diff
@@ -361,11 +362,7 @@ static void connected(struct bt_conn *conn, uint8_t conn_err)
 			bt_conn_unref(default_conn);
 			default_conn = NULL;
 
-			err = bt_scan_start(BT_SCAN_TYPE_SCAN_ACTIVE);
-			if (err) {
-				LOG_ERR("Scanning failed to start (err %d)",
-					err);
-			}
+			k_work_submit(&scan_work);
```

Comment:
Consider `(void)` cast to explicitly denote that we ignore the returned value (also in other places)